src/applications/facility-locator/components/search-results/LocationDirectionsLink.jsx

```diff
@@ -15,7 +15,19 @@ class LocationDirectionsLink extends Component {
       address = `${lat},${long}`;
     }
 
-    return (
+    return location.resultItem ? (
+      <dd>
+        <a
+          href={`https://maps.google.com?saddr=Current+Location&daddr=${address}`}
+          rel="noopener noreferrer"
+          target="_blank"
+          style={{ textDecoration: 'underline' }}
+        >
+          {from === 'FacilityDetail' && <i className="fa fa-road" />}
```

Comment:
this has been here before ^

Reply:
Instead of the `from === 'FacilityDetail'` comparison, the `FacilityDetail ` component itself could pass a boolean -

```
<LocationDirectionsLink isFacilityDetail>
```